### ISSUE TITLE
Always pass symbols to the API as uppercase

### ISSUE THREAD

**Issue by alexgolec** [alexgolec/tda-api#175]
Some endpoints require uppercase and some tolerate lowercase, so let's always pass uppercase to make things easier.

**Comment by pssolanki111** [comment]
is it still not implemented?

**Comment by alexgolec** [comment]
Nope, not yet. I've been busy with other things. You're welcome to submit a PR if you like.

**Comment by pssolanki111** [comment]
alrightyy. My current fork of `tda-api` is messed up so I'll have to remove and re-fork it. I'll raise a PR if I can find the time : )
